### DISCUSSION TITLE
Quasar, Electron, q-file, how can user specify a new file?

### DISCUSSION THREAD

**Discussion by MichaelJCole** [quasarframework/quasar#15013] [CLI - Electron mode] [answered]
Hi, thanks for reading.

I'm using Quasar in Electron and would like a feature that saves a file.  What's the best way to specify that file?

q-file can pick existing files and directories, but I didn't see a way to specify a filename for a new file.  What's the best way to do that?

**Comment by smithGit** [comment, marked as answer]
This may be one avenue - the Electron open dialogue:  https://www.electronjs.org/docs/latest/api/dialog
